client-sdk/go/client/model_orphan_volume_delete_result.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2023 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package swagger
+
+type OrphanVolumeDeleteResult struct {
```

Comment:
This is not consistent with the spec. Can you generate it again?

Reply:
Regenerated the client-sdk from latest spec.